signin.js

```diff
@@ -0,0 +1,84 @@
+const emailInput= document.querySelector('.email-input');
+const pwdInput=document.querySelector('.pwd-input');
+const loginButton=document.querySelector('.button-sign');
+const checkEmail= document.querySelector('.check-email');
+const checkPwd= document.querySelector('.check-pwd');
+const eyeIcon=document.querySelector('.pwd-eye-off');
+
+function emailCheck(email_address) {		
+    const email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
+    if (!email_regex.test(email_address)) return false; 
+    else return true;
+}
+
+eyeIcon.addEventListener("click", (e) => {
+    e.preventDefault();
+ });
+
+function loginCheck() {
+    if ((emailInput.value==='test@codeit.com')&&pwdInput.value==='codeit101') {
```

Comment:
1. 적당히 띄어쓰기 사용해주세요! 
`===` 와 같은 비교연산자도 좌우로 한 칸씩 띄는 것이 일반적입니다 

2. `test@codeit.com`, `codeit101` 와 같이 항상 값이 고정되어있는 것을 '상수' 라고 합니다
상수가 코드 안에 섞이게 되면 이후 상수를 변경해야 할 때 찾기 힘들기도 하고 관리가 어려워서 
보통 상수는 별도로 분리합니다. 
상수는 대문자 + snake_case 로 작성합니다 -> `SNAKE_CASE` <- 이렇게

예를 들어, 
```
const VALID_EMAIL = 'test@codeit.com';
const VALID_PASSWORD = 'codeit101';
```
와 같이 파일 상단 또는 별도 파일에 작성해두고 여기서는 
```suggestion
    emailInput.value === VALID_EMAIL &&
    passwordInput.value === VALID_PASSWORD
```

로 작성할 수 있습니다!

(참고로 여기서 valid 라는 단어는 '유효한', '통과 가능한' 의 의미로 사용했습니다)